### ISSUE TITLE
nrf52dk/nordic_soft_device: not working anymore?

### ISSUE THREAD

**Issue by PeterKietzmann** [RIOT-OS/RIOT#6379]
Following [this guide](https://devzone.nordicsemi.com/blogs/981/setting-up-ipv6-over-ble-using-nrf52-series-and-ri/) I can not use the nrf52dk with Nordic soft device BLE support any more. A program including the soft device module won't even allow interaction with a shell. @kaspar030 can you confirm?

**Comment by PeterKietzmann** [comment]
@kaspar030 ping can you give it a quick try?

**Comment by kYc0o** [comment]
I tried and it actually doesn't work anymore, it works for release 2016.10 but it stopped to work on 2017.01.

What I suggest is just to remove it from the boards with radio capabilities and point to the exact branch/commit on which the softdevice used to work.

Of course, the best thing to do is to fix it, but I have no idea how.

**Comment by triple7** [comment]
Hi, has anyone been able to fix this? We
re currently stuck with trying to get it going, however we don't want to mess with the dependencies all over the board.

**Comment by kaspar030** [comment]
I'll take a look at it later today!

**Comment by triple7** [comment]
Any progress on this?

**Comment by aabadie** [comment]
ping @kaspar030, would be great to fix this for the release.

**Comment by R1oT01** [comment]
@kaspar030 PING. Any ideas yet?

**Comment by R1oT01** [comment]
Ok, I tested some things, but still no output in terminal.
I flashed the SoftDevice.hex and microcoap-server manually with JLinkExe and nrfjprog command line tool, but still didn't work. I try to debug, but couldn't connect with the Segger JLink Debugger Device to the board. I tried to search the code for inconsistancies, but was overwhelmed by the amount of code to look into. Still have no clue where to search for the problem. Besides that, I'm a little frustrated about the fact, that i got no reaction yet. I'm a rookie in programming, especially with RIOT and Nordic devices. I would appreciate a little help.
